Purchases/Networking/Operations/Handling/SubscriberAttributeHandler.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SubscriberAttributeHandler.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+class SubscriberAttributeHandler {
+
+    let userInfoAttributeParser: UserInfoAttributeParser
+
+    init(userInfoAttributeParser: UserInfoAttributeParser = UserInfoAttributeParser()) {
+        self.userInfoAttributeParser = userInfoAttributeParser
+    }
+
+    func handleSubscriberAttributesResult(statusCode: Int,
+                                          maybeResponse: [String: Any]?,
+                                          maybeError: Error?,
+                                          maybeCompletion: PostRequestResponseHandler?) {
+        guard let completion = maybeCompletion else {
```

Comment:
nit: in other places, we check for nil completion before calling into the handler

Reply:
Good point, updating!